protocol/0093-CFUT-product_builtin_capped_future.md

```diff
@@ -0,0 +1,63 @@
+# Built-in [Product](./0051-PROD-product.md): Capped future
```

Comment:
Note that this is expected to be implemented by adding two parameters two the standard Future product. These parameters will be such that leaving them out (i.e. setting them to default values) will result in a standard future.

There won't be a separate product. It would make sense to rework the content in this file to align with this approach.